src/utils/directoryListing.ts

```diff
@@ -0,0 +1,123 @@
+import Handlebars from 'handlebars';
+import type { File, ReadDirectoryResult } from '../providers/provider';
+import htmlTemplate from '../templates/directoryListing.out';
+import { toReadableBytes } from '../utils/object';
+
+const handlebarsTemplate = Handlebars.template(htmlTemplate);
+
+const months = [
+  'Jan',
+  'Feb',
+  'Mar',
+  'Apr',
+  'May',
+  'Jun',
+  'Jul',
+  'Aug',
+  'Sep',
+  'Oct',
+  'Nov',
+  'Dec',
+];
+
+/**
+ * Render a directory listing
+ *
+ * TODO: Work with other teams on removing their dependency on nginx's listing
```

Comment:
Can you actually create an issue on this repo to keep track of this?

Reply:
#131